lib/tic_tac_toe.play.ex

```diff
@@ -0,0 +1,7 @@
+defmodule Mix.Tasks.Play do
+  use Mix.Task
+
+  def run(_) do
```

Comment:
Very nice! Does this mean you just have to type `mix play` to play your game?

Reply:
Indeed!

Reply:
+1